sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_readonlyfields.json

```diff
@@ -3039,11 +3039,13 @@
                 "status": {
                     "format": "int32",
                     "description": "Method invocation result status.",
-                    "type": "integer"
+                    "type": "integer",
+                    "readOnly": true
                 },
                 "payload": {
                     "description": "Method invocation result payload.",
-                    "type": "object"
+                    "type": "object",
+                    "readOnly": true
```

Comment:
For my understanding, we are setting these to readOnly because these properties are only contained in Response objects, correct?

Reply:
That's correct!